__tests-pacts__/index.ts

```diff
@@ -354,6 +354,22 @@ const pactSpec: PactSpec = {
       ],
     ],
   },
+  PageQuery: {
```

Comment:
```suggestion
  PagesQuery: {
```
It has to be the same name as in DB Layer, otherwise it won't have communication between both.